### DISCUSSION TITLE
统计下载需求

### DISCUSSION THREAD

**Discussion by anobaka** [anobaka/Bakabase#310] [Ideas]
为完成更通用的下载器，需要统计大家的下载需求。
目前根据数据结构 #309 ，已支持：

| 站点     | 类型                                 | 自动更新间隔 | 支持页码范围 | 检查点 |
|----------|--------------------------------------|--------------|--------------|--------|
| EH       | 单个作品                             | 自定义       | ×            |        |
| EH       | 关注列表                             | 自定义       | ×            |        |
| EH       | 搜索结果                             | 自定义       | √            |        |
| Pixiv    | 列表(标签、画师、搜索结果、收藏列表) | 自定义       | √            |        |
| Pixiv    | 排行榜                               | 仅限每日     | ×            |        |
| Pixiv    | 关注列表                             | 仅限每日     | √            |        |
| Bilibili | 收藏夹                               | 自定义       | ×            | aid    |
|          |                                      |              |              |        |
|          |                                      |              |              |        |

希望大家可以提出不在这个列表内的下载需求，站点也不仅限于EH和Pixiv

**Comment by liuhanqing-phy** [comment]
调用[qBittorrent](https://qbittorrent.org/)或者其他下载软件，从[动漫花园](https://share.dmhy.org/)或者[蜜柑计划](https://mikanani.me/)用磁力链下载动画的功能，不知道大佬愿不愿意做一下

**Reply by anobaka** [reply]
暂时不内置bt、磁链和一般下载功能，该类功能从0做到0.9很容易，从0.9做到1成本非常高。
请暂时先使用成熟的软件完成该类下载任务

**Comment by anobaka** [comment]
合并至 https://github.com/anobaka/InsideWorld/discussions/506
